advanced/lighttpd.conf.debian

```diff
@@ -59,21 +60,9 @@ $HTTP["url"] =~ "^/admin/" {
         "X-Pi-hole" => "The Pi-hole Web interface is working!",
         "X-Frame-Options" => "DENY"
     )
-}
-
-# Rewite js requests, must be out of $HTTP block due to bug #2526
-url.rewrite = ( "^(?!/admin/).*\.js$"  => "pihole/index.js"   )
-
-# If the URL does not start with /admin, then it is a query for an ad domain
-$HTTP["url"] =~ "^(?!/admin)/.*" {
-    # Create a response header for debugging using curl -I
-    setenv.add-response-header = ( "X-Pi-hole" => "A black hole for Internet advertisements." )
```

Comment:
IIRC it's replaced by [this codeblock](https://github.com/WaLLy3K/pi-hole/blob/78143c2ff41c2088b7db9efbeb882a62de944ef3/advanced/index.php#L46).